src/main/java/com/revature/batchservice/controller/BatchController.java

```diff
@@ -0,0 +1,121 @@
+package com.revature.batchservice.controller;
+
+import java.util.List;
+
+import org.jboss.logging.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.revature.batchservice.entity.BatchEntity;
+import com.revature.batchservice.service.BatchService;
+
+/**
+ * The BatchController is responsible for handling request about batches. It is mapped to CaliberProjectURL/batch.
+ * The BatchController can return a list of batches, a batch by Id.  It can add to, update, and delete a batch from our database.
+ * The BatchController expects request in JSON and send responses in JSON.
+ * 
+ * @author Justin Tu, Bita Mahbod, Daniel Mitre
+ *
+ */
+@RestController()
+@RequestMapping("/batch")
+@CrossOrigin
+public class BatchController {
+
+	private Logger log = Logger.getLogger("BatchController");
+	
+	@Autowired
+	private BatchService bs;
+	
+	/**
+	 * Accepts a HTTP GET request. 
+	 * Returns a List of all BatchEntities on the connected database as a JSON object.
+	 * @return a List<BatchEntity> that contains all BatchEntities in the database.
+	 */
+	@GetMapping()
+	public List<BatchEntity> getAllBatches() {
+		log.debug("Inside getAllBatches");
+		System.out.println("Inside getAllBatches");
+		return bs.findAllBatches();
+	}
+	
+	/**
+	 * Accepts a HTTP GET request. Mapped to ProjectURL/id/{pathVariable}
+	 * Returns a BatchEntity as a JSON entity based on the given id.
+	 * @param id An Integer that contains the BatchEntity id. The id is passed as a path variable. 
+	 * @return a BatchEntity which has the same id as the given id. Null if no matching 
+	 * 			id was found. Value is returned as a JSON object.
+	 */
+	@GetMapping("/id/{id}")
+	public BatchEntity getBatchById(@PathVariable("id") Integer id) {
+		log.debug("Inside findBatchById");
+		System.out.println("Inside findBatchById");
```

Comment:
Remove sysouts, logs should be sufficient

Reply:
Newest commit implements your changes.